session.go

```diff
@@ -475,6 +475,21 @@ func (s *Session) SetTrace(trace Tracer) {
 	s.mu.Unlock()
 }
 
+var selectWildcardNextWarning time.Time
+var mu sync.Mutex
+
+func warnSelectWildcardStartPerformanceIssue() {
+	mu.Lock()
+	defer mu.Unlock()
+
+	now := time.Now()
+
+	if now.Sub(selectWildcardNextWarning) >= time.Minute {
+		selectWildcardNextWarning = now
+		Logger.Println("warning: skipping metadata is disabled for `SELECT * FROM ...` queries due to potential correctness issues when altering the table. This may cause a decrease in performance. It is recommended to avoid using `SELECT * FROM ...` and provide specific column names.")
```

Comment:
https://github.com/search?q=repo%3Ascylladb%2Fgocql+%22SELECT+*+%22&type=code ...

Quoting Phil Collins: 'Just do as I say, don't do as I do' ;-)

Reply:
Schema queries are issued relatively rarely, and often return many rows.
The overhead is bigger if there is very small number of rows returned - because then the metadata returned is a bigger part of the whole response.

Reply:
indeed, I think I've opened an issue about it somewhere.